lib/rdoc/markup/to_rdoc.rb

```diff
@@ -3,6 +3,14 @@
 # Outputs RDoc markup as RDoc markup! (mostly)
 
 class RDoc::Markup::ToRdoc < RDoc::Markup::Formatter
+  DEFAULT_HEADINGS = {
+      1 => ['= ',      ''],
+      2 => ['== ',     ''],
+      3 => ['=== ',    ''],
+      4 => ['==== ',   ''],
+      5 => ['===== ',  ''],
+      6 => ['====== ', '']
+  }.tap { |h| h.default = [] }.freeze
```

Comment:
JFYI: I like the following style (no `tap`, no `freeze`) but it's just my preference. It's not required for merging.

```suggestion
  }
  DEFAULT_HEADINGS.default = []
```

Reply:
I think freezing a mutable structure (e.g. array, hash...etc.) constant is still a recommended practice if we don't expect it to change?

Reply:
I know some people recommend the style but I think that I don't like the style because:

* Many `.freeze` codes are (a bit) ugly
* Constant (`XXX = {}`) already shows our intention (we don't expect it to change)

(NOTE: I don't object the style in RDoc.)

Reply:
I tried meeting everybody half way and split the default and freezing into separate lines.